itou/www/siae_evaluations_views/forms.py

```diff
@@ -302,3 +224,69 @@ def clean(self):
             if cleaned_data.get("permanent_suspension"):
                 cleaned_data["suspension_dates"] = InclusiveDateRange(cleaned_data["permanent_suspension"])
         return cleaned_data
+
+
+class InstitutionEvaluatedSiaeNotifyStep3EvaluatedJobApplicationFormSet(forms.BaseFormSet):
+    def __init__(self, sanctions, evaluated_job_applications_to_sanction, *args, **kwargs):
+        self.evaluated_job_applications_to_sanction = {
+            job_app.pk: job_app for job_app in evaluated_job_applications_to_sanction
+        }
+        initial = (
+            [
+                {
+                    "evaluated_job_application": job_app.pk,
+                    "subsidy_cut_percent": None,
+                }
+                for job_app in self.evaluated_job_applications_to_sanction.values()
+            ]
+            if SUBSIDY_CUT in sanctions
+            else []
+        )
+        super().__init__(*args, initial=initial, **kwargs)
+
+    def clean(self):
+        """Check that at least one evaluated job evaluation has a non zero percentage"""
+        if any(self.errors):
+            return
+
+        for form in self.forms:
+            if form.cleaned_data.get("subsidy_cut_percent"):
+                return
+        raise ValidationError(
+            "Un pourcentage compris entre 0 et 100 doit être appliqué sur chacune des "
+            "auto-prescriptions listées ci-dessous."
```

Comment:
```suggestion
            "Un pourcentage compris entre 0 et 100 doit être appliqué sur au moins l'une des "
            "auto-prescriptions listées ci-dessous."
```

Pour être cohérent avec la docstring et la boucle qui short circuit